src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.Node.cs

```diff
@@ -101,6 +102,7 @@ private string GetDebuggerDisplay()
             /// - Primitive types
             /// - Types which is not a generic method parameter
             /// - By reference type of any types above
+            /// - Array types of with element of any types above
             /// </summary>
             public readonly bool IsComplexType;
```

Comment:
if IsArrayType implies IsComplexType, should we use an enum instead?

Reply:
No, array doesn't indicate it's complex type, it depends on the element of the array

Reply:
ah... that wasn't totally clear to me from the docs (though they docs are correct).  perhaps explicitly call out that simple array types (i.e. Int32[]) wouldn't be complex.